packages/core/src/rest/rest-request-config.ts

```diff
@@ -0,0 +1,9 @@
+export class RestRequestConfig {
```

Comment:
This concept of the odata request builders was there because we potentially construct a request (with top, skip, orderby, filter, etc.).
Here it is not really needed and seems like overkill to me.
I see two options:
1. I think it would be acceptable if we base the requestbuilder on the methodrequestbuilderbase (that would need some changes to be independent of odata). So probably add a layer below methodrequestbuilderbase (and maybe rename it to odatarequestbuilder).
2. Just store this in the request builder